packages/protocol/migrations_sol/constants.sol

```diff
@@ -3,6 +3,9 @@ pragma solidity >=0.8.7 <0.8.20;
 import { TestConstants } from "@test-sol/constants.sol";
 
 contract MigrationsConstants is TestConstants {
+  // Addresses
+  address constant DEPLOYER_ACCOUNT = 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266;
```

Comment:
this is not really a constant (this also applies to `constants.sh` but I don't think we need to refactor that now).

It's usually a bad practice to have code as configurations (vs a json file or a parameter), at some point users should be able to specify the deployer account without having to recompile the solidity. From the anvil docs looks like the only option is a json file.

Reply:
Understood, on that basis, I'll leave this as is.